modules/t/test-genome-DBs/multi/ontology/meta.txt

```diff
@@ -59,3 +59,4 @@
 62	patch	patch_112_113_a.sql|schema_version	\N
 63	patch	patch_112_113_b.sql|Ensure meta_value is not null	\N
 64	patch	patch_113_114_a.sql|schema_version	\N
+65	patch	patch_114_115_a.sql|schema_version	\N
```

Comment:
Hi @jamie-m-a , do we want to add the same changes to core, variation and otherfeatures?

- `modules/t/test-genome-DBs/homo_sapiens/core/meta.txt`
- `modules/t/test-genome-DBs/homo_sapiens/variation/meta.txt`
- `modules/t/test-genome-DBs/homo_sapiens/otherfeatures/meta.txt`

Reply:
Technically I guess, they don't seem to get done by the script, so I imagine they're not actually important for our tests. But I'll add them for consistency

Reply:
Odd. Will add.

Reply:
Variation is already present as per Diana's changes, I've added for core and otherfeatures.